__tests__/app.test.js

```diff
@@ -41,6 +41,58 @@ describe("/api/topics", () => {
   });
 });
 
+describe("/api/articles/:article_id", () => {
+  test("GET 200: Responds with a single article by its id", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then(({ body }) => {
+        const { article } = body;
+        const {
+          author,
+          title,
+          article_id,
+          topic,
+          created_at,
+          votes,
+          article_img_url,
+        } = article;
+        expect(author).toBe("butter_bridge");
+        expect(title).toBe("Living in the shadow of a great man");
+        expect(article_id).toBe(1);
+        expect(article.body).toBe("I find this existence challenging");
+        expect(topic).toBe("mitch");
+        // TO FIX THE BELOW FAILED TEST:
+        // Expected: "2020-07-09T21:11:00.000Z"
+        // Received: "2020-07-09T20:11:00.000Z"
+        // expect(created_at).toBe(new Date(1594329060000).toISOString());
+        expect(typeof created_at).toBe("string");
+        expect(votes).toBe(100);
+        expect(article_img_url).toBe(
+          "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700"
+        );
```

Comment:
Another way to do this is by using .toMatchObject or .objectContaining. This would be a more concise and robust way to assert the structure of each topic object. Here is the link for toMatchObject - https://jestjs.io/docs/expect#tomatchobjectobject

Remember to remove any comments from your code 👀